lib/formatador/table.rb

```diff
@@ -80,7 +80,11 @@ def display_compact_table(hashes, keys = nil, &block)
   private
 
   def length(value)
-    value.to_s.gsub(PARSE_REGEX, '').length
```

Comment:
Hey, sorry for the delay, I was still pondering stuff. What if we wrapped this with something like:
```
if const_defined?(:Unicode)
```

That way, you wouldn't be required to install unicode, but if you did (and required it before formatador) it would work. What do you think?